src/pages/workspace/withFullPolicy.js

```diff
@@ -73,9 +72,6 @@ const fullPolicyPropTypes = {
         */
         errorFields: PropTypes.objectOf(PropTypes.objectOf(PropTypes.string)),
     }),
-
-    /** The policy member list for the current route */
-    policyMemberList: PropTypes.objectOf(policyMemberPropType),
```

Comment:
Why is this removed?

Reply:
Damn, good question. If I have to be honest, I don't quite recall... 🤔

Reply:
Oh I remember! Because the member list is now it's own key, not inside the policy anymore

Reply:
They already separate key along with policy

Reply:
@iwiznia

Reply:
Yep, that's what I said too 😄